shared/chat/conversation/container.js

```diff
@@ -39,15 +41,16 @@ const mapStateToProps = (state: TypedState, {routePath, routeState}) => {
   let threadLoadedOffline = false
 
   if (selectedConversationIDKey !== Constants.nothingSelected) {
+    rekeyInfo = state.chat.get('rekeyInfos').get(selectedConversationIDKey)
```

Comment:
we know this now, also if we can't load the conversation we still want to know we're in this state